### PR TITLE
Eduardo

### PR DESCRIPTION
**Pull request by helloeduardo**
Completed iteration 3

Didn't have time to complete iteration 4, specifically:
- 1. attending the most expensive of two exhibits
- 2. testing patrons of exhibits was not completed
- 3. did complete revenue, but test didn't pass to spec due to issue 1, resulting in 5 less dollars

### CODE REVIEW

File: test/museum_test.rb [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+    patron_2.add_interest("Dead Sea Scrolls")
+
+    patron_3 = Patron.new("Johnny", 5)
+    patron_3.add_interest("Dead Sea Scrolls")
+
+    dmns.admit(patron_1)
+    dmns.admit(patron_2)
+    dmns.admit(patron_3)
+
+    dmns.draw_lottery_winner(dead_sea_scrolls) #Need to assert random patron name
```
Comment (helloeduardo) [inline test/museum_test.rb:171]: Tried `assert dmns.draw_lottery_winner(dead_sea_scrolls) == ("Bob" || "Johnny")` but that didn't work. Is there a mintiest assertion for THIS OR THAT?

Comment (helloeduardo) [reply test/museum_test.rb:171]: NVM, I thought about it more! I needed to `assert (dmns.draw_lottery_winner(dead_sea_scrolls) == "Bob" || dmns.draw_lottery_winner(dead_sea_scrolls) == "Johnny")`

Comment (KathleenScriver) [reply test/museum_test.rb:171]: There is also an assertion you can use that's `assert_includes ["Bob", "Johnny"], dmns.draw_lottery_winner(imx)` that would work too

**Comment by KathleenScriver** [comment]
Great job with your commits

**Review by KathleenScriver — COMMENTED** [review]
You have done a fantastic job here. Great use of specific enumerables and great working with hashes. Have you explored the #reduce enumberable yet? I encourage you to get some practice with that one as well. Keep up the great work.

**Pull request closed without merging**